File: android/versioned-abis/expoview-abi34_0_0/src/main/java/abi34_0_0/expo/modules/imagemanipulator/ImageManipulatorPackage.java
package abi34_0_0.expo.modules.imagemanipulator;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi34_0_0.org.unimodules.core.BasePackage;
import abi34_0_0.org.unimodules.core.ExportedModule;

public class ImageManipulatorPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new ImageManipulatorModule(context));
  }
}

File: android/versioned-abis/expoview-abi42_0_0/src/main/java/abi42_0_0/expo/modules/imagemanipulator/ImageManipulatorPackage.kt
package abi42_0_0.expo.modules.imagemanipulator

import android.content.Context
import abi42_0_0.org.unimodules.core.BasePackage

class ImageManipulatorPackage : BasePackage() {
  override fun createExportedModules(context: Context) = listOf(ImageManipulatorModule(context))
}
